### ISSUE TITLE
Cannot tab past first calendar in lefthand sidebar

### ISSUE THREAD

**Issue by miaulalala** [nextcloud/calendar#5867]
### Steps to reproduce

1. Try tabbing the app navigation
2. Navigation indicator is stuck on first calendar in list.

### Expected behavior

I can tab to the end

### Actual behaviour

I'm stuck on the first calendar in the list

### Calendar app version

4.7.0-beta.3

### Browser

FF 124

### Client operating system

Ubunut 20

### Server operating system

Ubunut 20

### Web server

Apache

### Database engine version

MySQL

### PHP engine version

PHP 8.2

### Updated from an older installed version or fresh install

None

### Browser log

```Text
[Vue warn]: Error in v-on handler: "TypeError: this.$refs.actions.$refs.menuButton is undefined"

found in

---> <NcAppNavigationItem>
       <CalendarListItem> at src/components/AppNavigation/CalendarList/CalendarListItem.vue
         <Draggable>
           <CalendarList> at src/components/AppNavigation/CalendarList.vue
             <NcAppNavigation>
               <NcContent>
                 <Calendar> at src/views/Calendar.vue
                   <App> at src/App.vue
                     <Root> vue.runtime.esm.js:4625
    VueJS 63
    addCalendar calendars.js:71
    wrappedMutationHandler vuex.esm.js:844
    commitIterator vuex.esm.js:466
    commit vuex.esm.js:465
    _withCommit vuex.esm.js:624
    commit vuex.esm.js:464
    boundCommit vuex.esm.js:409
    loadCollections calendars.js:680
    loadCollections calendars.js:679
    wrappedActionHandler vuex.esm.js:851
    dispatch vuex.esm.js:516
    boundDispatch vuex.esm.js:406
    beforeMount Calendar.vue:183
    VueJS 27
    <anonymous> main.js:72
    <anonymous> calendar-main.js:307258
    <anonymous> calendar-main.js:307260
```
